test/unit/ngsiv2/provisioning/removeProvisionedDevice-test.js

```diff
@@ -231,7 +231,42 @@ describe('NGSI-v2 - Device provisioning API: Remove provisioned devices', functi
             method: 'DELETE'
         };
 
-        it('should return a 200 OK and no errors', function (done) {
+        it('should return a 204 OK and no errors', function (done) {
+            request(options, function (error, response, body) {
+                should.not.exist(error);
+                response.statusCode.should.equal(204);
+                done();
+            });
+        });
+    });
+
+    describe('When a request to remove a provision devices arrives', function () {
```

Comment:
Should functional test (i.e. the ones within /test/functional/) to be included?

Reply:
Not sure if there is support for northbound api in functiona tests. @mapedraza is there support for that ?

Reply:
So far, it is not supported northbound only tests (the tests consits of 3 steps, group provision, measure send and CB validation). No so hard to add support adding another "type" of test that check only the northbound using the same test infrastructure. I suggest to merge this like this and create an issue to implement what is describe in this comment.

Reply:
Issue created: https://github.com/telefonicaid/iotagent-node-lib/issues/1581 (please feel free to comment on it if something is missing/wrong)

NTC